query-old/docs/api/created-api/hooks.md

```diff
@@ -91,18 +91,40 @@ The query arg is used as a cache key. Changing the query arg will tell the hook
 #### Signature
 
 ```ts
-type MutationHook = () => [
-  (
-    arg: any
-  ) => {
-    unwrap: () => Promise<ResultTypeFrom<D>>;
-  },
-  MutationSubState<D> & RequestStatusFlags
-];
+type UseMutation<Definition> = () => [UseMutationTrigger<Definition>, UseMutationResult<Definition>];
+
+type UseMutationTrigger<Definition> = (
+  arg: ArgTypeFrom<Definition>
+) => Promise<{ data: ResultTypeFrom<Definition> }> & {
+  arg: {
+    endpointName: string; // The name of the given endpoint for the mutation
+    originalArgs: ArgTypeFrom<Definition>; // Arguments passed to the mutation
+    track?: boolean; // Whether the mutation is being tracked in the store
+    startedTimeStamp: number; // Timestamp for when the mutation was initiated
+  };
```

Comment:
This is kinda "implementation detail". Could we mark that as `@internal` in the docblock and omit it here instead?

Reply:
Yep sounds good to me